proto/ory/keto/acl/v1alpha1/check_service.proto

```diff
@@ -73,6 +73,8 @@ message CheckRequest {
   // ACLs had already been replicated to all availability zones.
   // -->
   string snaptoken = 6;
+  // The maximum depth to search for a relation.
```

Comment:
It would be nice to add a short text about the default value here.

Reply:
Agreed. Adding